### PR TITLE
Support Landing Page - Part 5 - Subscribe Tweaks

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

The subscribe section was only half-finished before, missing the correct copy and links etc. This brings it in line with the functionality of the subscribe bundles on the current landing page.

[**Trello Card**](https://trello.com/c/pADcFz7O/1093-support-landing-subscribe-tweaks)

cc: @Amohkhan

## Changes

- Connected it to redux to retrieve all the information that needs to be passed through to the subs site in the URL.
- Updated the copy of the bundles to display lists of features, with ticks!
- Updated the CTA labels.
- Added the links on the CTAs, with the same functionality as the current bundles page, i.e. promo codes, acquisition data etc.
- Created a helper to manage the lists of subs bundle features.
- Added a background colour to differentiate from the contribute section in #366.
- Tweaked the fonts.

## Screenshots

**Desktop:**

subs-desktop

**Mobile:**

subs-mobile

### CODE REVIEW

File: assets/pages/bundles-landing/support-landing-ab-test/components/subscriptionBundleNewDesign.jsx [hunk 1]
```diff
@@ -14,9 +17,10 @@ type PropTypes = {
   heading: string,
   price: string,
   from: boolean,
-  copy: string,
+  copy: ListItem[],
```
Comment (davidfurey) [inline assets/pages/bundles-landing/support-landing-ab-test/components/subscriptionBundleNewDesign.jsx]: Maybe not for this PR, but I find it very confusing when we use the word "copy" to mean "text supplied by editorial".  Maybe in this context `description` might be clearer?

File: assets/pages/bundles-landing/support-landing-ab-test/supportLanding.scss [hunk 2]
```diff
@@ -291,25 +291,38 @@
...
 
     .subscription-bundle__copy {
-      font-family: $gu-text-egyptian-web;
-      font-size: 18px;
-      line-height: 24px;
       color: gu-colour(neutral-2);
       flex-grow: 1;
+
+      .component-feature-list {
+        list-style-image: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 28 28"><style>.st0{fill:#333;}</style><path class="st0" d="M7.885 25.74L0 14.82l3.22-3.217L8.3 16.74 24.695 2.26 28 5.565"/></svg>');
```
Comment (svillafe) [inline assets/pages/bundles-landing/support-landing-ab-test/supportLanding.scss]: Is there a way to avoid this hardcoded SVG?

Comment (Ap0c) [reply assets/pages/bundles-landing/support-landing-ab-test/supportLanding.scss]: This is a problem I hit first time round with the bundles landing page (this is the same solution). It's difficult to do it in a way that works cross-browser. Last time I ended up with alignment issues all over the place depending on the browser you open the page with (and the SVG doesn't show at all in IE10).

It's possible that Amo is going to give me a new SVG for this, and if he does I'll see if I can come up with another solution, but I'm not 100% confident 😐.

File: assets/pages/bundles-landing/support-landing-ab-test/components/subscriptionBundleNewDesign.jsx (hunk 1 above)
Comment (Ap0c) [reply assets/pages/bundles-landing/support-landing-ab-test/components/subscriptionBundleNewDesign.jsx]: I understand, I can change it to 'benefits' if that helps? Description puts me more in an a11y mindset.

The word 'copy' is almost always used by design and UX when talking about this type of content, so it's become my word for it too 🤓.

Comment (davidfurey) [reply assets/pages/bundles-landing/support-landing-ab-test/components/subscriptionBundleNewDesign.jsx]: Benefits is fairly clear to me.  You should stop hanging out with design & UX, they are clearly a bad influence.

**Pull request merged**